### ISSUE TITLE
Webserve link is not working

### ISSUE THREAD

**Issue by mbani01** [benlyazid/benlyazid.github.io#1]
[webserve link](https://github.com/yesiamdollar/WebServ) isn't working, I guess it's a private repo

**Comment by benlyazid** [comment]
Yes as you said it's a private repo I will fork the repo and put the new link.
Thanks for reporting I really appreciate it.
